README.md

```diff
@@ -131,6 +131,9 @@ kube-system                kubernetes-dashboard-5f7b999d65-ntrxb         1/1
 ```
 
 2. Create a cluster:
+
+You have to [inspect](docs/ibmcloud_help.md#inspect-files-clusteryaml-and-machinesyaml) files `cluster.yaml`, `machines.yaml` before create a cluster on IBM Cloud.
```

Comment:
- `s/create/creating/`
 - `s/have to/should/`
 - `inspect` is a verb, a link should be better for a noun word

Reply:
Update for doc review.